### DISCUSSION TITLE
Bluepy в HA 2022.07

### DISCUSSION THREAD

**Discussion by fishenebelny** [TionAPI/HA-tion#86] [Q&A] [answered]
Я правильно понял, что компонент управления тионами не будет работать в последней версии HA, так как bluepy уже не поддерживается в нем?

**Comment by IATkachenko** [comment, marked as answer]
Работать, видимо, продолжит, поскольку `bluepy` указан в зависимостях модулей.

Но уххх, какого слона я пропустил....
Они, НАКОНЕЦ, перешли на `bleak`!!!! 
Это решит 99% наших проблем :) 
К `bleak` я почти готов. Может даже в выходные релиз сделаю.

**Reply by fishenebelny** [reply]
Bluepy, как я понял, не будет работать с Python 3.10.

**Reply by IATkachenko** [reply]
> Bluepy, как я понял, не будет работать с Python 3.10.

В любом случае, мы, наконец, сможем от него отказаться :) Меня ужерживало только то, что `bleak` не получалось протолнуть в HomeAssistant. Но сейчас это поправили и нет ни одной причины не перейти на `bleak`.
